channels/br.m3u

```diff
@@ -87,6 +87,8 @@ https://59f2354c05961.streamlock.net:1443/rbtv/_definst_/rbtv/playlist.m3u8
 http://rbc.directradios.com:1935/rbc/rbc_720p/live.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/5VtLILS.png" group-title="General",Rede ConecTV
 http://stream02.msolutionbrasil.com.br:1935/conectv/live/livestream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/OLYyZDD.png" group-title="General",Rede Minas
+https://v4-slbps-sambavideos.akamaized.net/live/3282%2C8114%2Cec4b5a296d97fa99bf990662f5b4f8e1%3Bbase64np%3BMc8VDxqNjXKCAf8%21/amlst%3AMc_tFgfGiHOdQXPB/chunklist_.m3u8
```

Comment:
The link is not working at the moment.

Reply:
Tested in Elmedia Video Player & VLC Player
<img width="317" alt="Captura de pantalla 2020-06-25 a las 14 39 41" src="https://user-images.githubusercontent.com/7253922/85713232-d713e400-b6f1-11ea-8e2e-a90699daa117.png">